### ISSUE TITLE
JS error when converting file

### ISSUE THREAD

**Issue by donmccurdy** [sbtron/makeglb#2]
This file renders correctly and passes validation, but when trying to use makeglb I get the error:

```
Uncaught TypeError: Cannot read property 'byteLength' of undefined
    at fileSave ((index):156)
    at checkRemaining ((index):136)
    at FileReader.reader.onload ((index):105)
```

[WaltHead.gltf.zip](https://github.com/sbtron/makeglb/files/1623212/WaltHead.gltf.zip)

**Comment by sjfricke** [comment]
I can confirm this is happening to me too

I spent a solid hour and cannot figure this out

First name a fake `.gltf` file and fill it with
```
{ 
 "buffers" : 
    [ {
      "uri" : "data:application/octet-stream;base64,AAABAAIAAAAAAAAAAAAAAAAAAAAAAIA/AAAAAAAAAAAAAAAAAACAPwAAAAA=",
      "byteLength" : 44
    },
    {
      "uri" : "data:application/octet-stream;base64,AAABAAIAAAAAAAAAAAAAAAAAAAAAAIA/AAAAAAAAAAAAAAAAAACAPwAAAAA=",
      "byteLength" : 44
    },
    {
      "uri" : "data:application/octet-stream;base64,AAAAAAAAgD4AAAA/AABAPwAAgD8AAAAAAAAAAAAAAAAAAIA/AAAAAAAAAAD0/TQ/9P00PwAAAAAAAAAAAACAPwAAAAAAAAAAAAAAAPT9ND/0/TS/AAAAAAAAAAAAAAAAAACAPw==",
      "byteLength" : 100
    } ]
}
```

and it fails, basically, the first `uri` is not getting into the `JSON.parse()` call for whatever reason

**BUT** if you do the smallest thing like

- change the **first** `uri` to `urii` it parses correctly
- change the `buffers` to `buffer` it parses correctly

I even did something like this
```
var e;

// ...

reader.onload = function(event) {
    e = new String(event.target.result);
    gltf = JSON.parse(e.toString());
    checkRemaining();
};
```
and did `JSON.parse(e.toString())` in the dev tool and it works there

**ALSO** tried a `setTimeout`... I honestly don't know anymore :'(

**Comment by donmccurdy** [comment]
@sjfricke think i have a fix in #4, could you test at https://makeglb-yujgfsaslr.now.sh/?

**Comment by sjfricke** [comment]
@donmccurdy I was looking in the wrong direction, but yes your fix works for me! Thanks again
